libs/s3/s3_tagging_test_lib.py

```diff
@@ -161,16 +161,18 @@ def set_object_tag(
         :return: (Boolean, response)
         """
         tag_count = kwargs.get("tag_count", 1)
+        tags = kwargs.get("tags", None)
         try:
-            LOGGER.info("Set object tagging")
```

Comment:
Removing lines 165 - 173 will break existing S3 tagging tests, which pass key/value pairs to the set_object_tag method. You can instead make the code block conditional and assign your "tags" from kwargs to tag_set and allow existing code to generate the tag set if "tags" is equal to None.

We will also need docstring to be updated.

Reply:
Noticed the new code block for handling this scenario, so no changes are needed. Resolving my comment